pairtools/lib/dedup.py

```diff
@@ -221,8 +279,16 @@ def _dedup_chunk(
     df = df.reset_index()  # Remove the index temporarily
 
     # Set up columns to store the dedup info:
-    df["clusterid"] = np.nan
-    df["duplicate"] = False
+    df.loc[:, "clusterid"] = np.nan
```

Comment:
@agalitsyna , tests fail at this line with an error message "ValueError: cannot set a frame with no defined index and a scalar".
Googling suggests that such an error arises when one tried to index an empty frame: https://stackoverflow.com/questions/48306694/valueerror-cannot-set-a-frame-with-no-defined-index-and-a-value-that-cannot-be

Reply:
ping @agalitsyna

Reply:
Maybe df.assign? What's your favorite way of filling in columns in dataframe? 
https://stackoverflow.com/questions/34811971/how-do-i-fill-a-column-with-one-value-in-pandas